consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/validation/RoundChangePayloadValidator.java

```diff
@@ -96,10 +96,11 @@ private boolean validateConsistencyOfPrepareCertificateMessages(
       return false;
     }
 
-    if (certificate.getPreparePayloads().size() < minimumPrepareMessages) {
+    if (certificate.getPreparePayloads().stream().map(SignedData::getAuthor).distinct().count()
```

Comment:
Do all the prepare authors also need to be unique? Or is it enough the we have minimumPrepareMessages. I guess I'm wondering if need a hasDuplicateAuthors check like in the RoundChangeCertificateValidator.